lightning-invoice/src/lib.rs

```diff
@@ -105,19 +143,19 @@ pub fn check_platform() {
     // If this test fails on your platform, please don't use the library and open an issue
     // instead so we can resolve the situation. Currently this library is tested on:
     //   * Linux (64bit)
-    let fail_date = UNIX_EPOCH + Duration::from_secs(SYSTEM_TIME_MAX_UNIX_TIMESTAMP);
-    let year = Duration::from_secs(60 * 60 * 24 * 365);
+	let fail_date = SystemTime::UNIX_EPOCH + Duration::from_secs(SYSTEM_TIME_MAX_UNIX_TIMESTAMP);
+	let year = Duration::from_secs(60 * 60 * 24 * 365);
 
-    // Make sure that the library will keep working for another year
-    assert!(fail_date.duration_since(SystemTime::now()).unwrap() > year);
+	// Make sure that the library will keep working for another year
```

Comment:
Could you clean up the whitespace (tabs instead of spaces) since you are touching this method?

Reply:
:+1: